### PR TITLE
update sidebar for import, update and export

### PR DESCRIPTION
**Pull request by wangtianyi2004**
## Versions 

- [x] dev
- [x] 3.0
- [x] 2.1
- [ ] 2.0

## Languages

- [x] Chinese
- [x] English

## Docs Checklist

- [ ] Checked by AI
- [ ] Test Cases Built

### CODE REVIEW

File: sidebars.json [hunk 1]
```diff
@@ -153,50 +153,50 @@
...
+                                "data-operate/import/data-source/huawei-obs",
+                                "data-operate/import/data-source/tencent-cos",
+                                "data-operate/import/data-source/minio",
+                                "data-operate/import/data-source/s3-compatible"
+                            ]
+                        },
+                        {
+                            "type": "category",
+                            "label": "Import Way",
+                            "items": [
```
Comment (KassieZ) [inline sidebars.json:179]: 确认顺序是否正确

File: sidebars.json [hunk 2]
```diff
@@ -153,50 +153,50 @@
...
+                                "data-operate/import/import-way/insert-into-manual",
+                                "data-operate/import/import-way/insert-into-values-manual",
+                                "data-operate/import/import-way/mysql-load-manual"
                             ]
                         },
+                        "data-operate/import/load-data-format",
+                        "data-operate/import/error-data-handling",
+                        "data-operate/import/load-data-convert",
+                        "data-operate/import/load-high-availability",
+                        "data-operate/import/migrate-data-from-other-olap",
```
Comment (KassieZ) [inline sidebars.json:192]: 是否在这里

File: sidebars.json [hunk 3]
```diff
@@ -220,19 +220,19 @@
...
-                            "label": "Exporting Data",
-                            "items": [
-                                "data-operate/export/export-overview",
-                                "data-operate/export/export-manual",
-                                "data-operate/export/outfile",
-                                "data-operate/export/export-with-mysql-dump"
-                            ]
-                        },
-                        "data-operate/transaction"
+			"data-operate/transaction"
```
Comment (KassieZ) [inline sidebars.json:223]: 是否在这里

File: sidebars.json [hunk 4]
```diff
@@ -220,19 +220,19 @@
                                 "data-operate/delete/table-temp-partition"
```
Comment (KassieZ) [inline sidebars.json:220]: 是否还在

File: sidebars.json (hunk 1 above)
Comment (wangtianyi2004) [reply sidebars.json:179]: 正确

Comment (wangtianyi2004) [reply sidebars.json:179]: 正确

**Review by wm1581066 — APPROVED** [review]
LGTM

**Pull request merged**